evalml/pipelines/pipeline_base.py

```diff
@@ -425,6 +426,58 @@ def feature_importance(self):
         df = pd.DataFrame(importance, columns=["feature", "importance"])
         return df
 
+    def graph_json(self):
```

Comment:
General question: what do you think about having this method *not* attached to the PipelineBase class? This feels like it could be its standalone util method that takes in a pipeline as an argument. Thoughts? 👀

Reply:
So my reasoning for putting this here was:
1. I was looking at this from the perspective of running `AutoML.search()` and then choosing the best pipeline and being able to make a quick function call to the `graph_json` to export it to a third party tool for visualization.
2. If the user has access to any pipeline, then calling this function would be just as easy of a step as passing it to a util function.
3. Just as `model_understanding` has functions broken up into calculations and visualization, even though `PipelineBase.graph` won't be using `graph_json`, having them together seemed appropriate.

I like your idea and I'm totally down for discussing it further. It just seems to me that since the function requires `component_dict`, it might be better to keep in in `PipelineBase`.